### PR TITLE
show hints while typing

### PR DESCRIPTION
**Pull request by WMDE-Fisch**

### CODE REVIEW

File: DeepCat.js [hunk 1]
```diff
@@ -44,21 +45,23 @@
 				'deepcat-error-tooldown': 'CatGraph-Tool is not reachable.',
 				'deepcat-error-unknown-graph': 'The Wiki is not supported by CatGraph.',
 				'deepcat-missing-category': 'Please insert a category.',
-				'deepcat-hintbox-close': 'Hide',
+				'deepcat-hintbox-close': 'hide in the future',
```
Comment (thiemowmde) [inline DeepCat.js]: Why not uppercase?

Comment (WMDE-Fisch) [inline DeepCat.js]: damn your fast ;-)

**Comment by wmde-manicki** [comment]
Apart from really minor suggestion on changing the button text, looks and works fine. I would merge it after rebasing.

**Comment by WMDE-Fisch** [comment]
PLZ DO NOT MERGE YET! 

We should clarify if we really want to have hints while typing before merging this.

**Comment by WMDE-Fisch** [comment]
can be merged when #37 is merged

**Comment by WMDE-Fisch** [comment]
can now also be merged @wmde-manicki / @jkroll20

**Pull request merged**